src/Ci/CiInterface.php

```diff
@@ -16,6 +17,11 @@ public static function isDetected(Env $env): bool;
      */
     public function getCiName(): string;
 
+    /**
+     * Return true if the current build is from a pull/merge request
```

Comment:
> Return true if the current build is from a pull/merge request

Does it? It returns the `TrinaryLogic`, not true, which suggest it returns bool.